### PR TITLE
Adds wasteland Cowboy profession and a Saloon Barfight Scenario

### PR DESCRIPTION
**Pull request by hyper2snyper**
```SUMMARY: Content "Adds wasteland Cowboy profession and a Saloon Barfight Scenario"```

#### Purpose of change
To add some cowboy stuff. related to #31320
#### Describe the solution
Added the wasteland cowboy profession that is useable on the next summer start.
#### Describe alternatives you've considered
not adding them

### CODE REVIEW

**Comment by I-am-Erk** [comment]
Would these be better suited to a mod? We already have so many professions.

**Comment by Rivet-the-Zombie** [comment]
>We already have so many professions.

I feel like there's no such thing as *too many* here. We just need a better means of sorting them during character creation.

**Comment by I-am-Erk** [comment]
That's true, although a lot of the non-mainstream ones could be sorted by putting them into mods IMO.

**Comment by Amoebka** [comment]
Next Summer scenario actually has only a few professions to choose from (4 or 5?), so adding new ones there is perfectly reasonable. Especially if the new profession doesn't show up in standart scenarios.

**Comment by tenmillimaster** [comment]
This profession makes so much more sense as a next-summer start+profession, but the scenario would need tweaking.

File: data/json/professions.json [hunk 1]
```diff
@@ -3743,5 +3743,80 @@
...
+     ],
+     "male": [ "boxer_shorts" ],
+     "female": [ "bra", "panties" ] 
+    }
+  },
+  {
+    "type": "profession",
+    "ident": "waste_cowboy",
+    "name": "Wasteland Cowboy",
+    "description": "",
```
Comment (John-Candlebury) [inline data/json/professions.json]: Seems like the description is missing

File: data/json/professions.json [hunk 2]
```diff
@@ -3743,5 +3743,80 @@
...
+          { "item": "454_Casull", "charges": 18, "container-item": "bandolier_pistol" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boy_shorts" ]
+    },
+    "flags": [ "SCEN_ONLY" ]
   }
-]
+]
```
Comment (ZhilkinSerg) [inline data/json/professions.json]:
```suggestion
]

```

**Comment by davidpwbrown** [comment]
Maybe for the desert mod? that needs some content :)

File: data/json/professions.json [hunk 3]
```diff
@@ -3743,5 +3743,80 @@
...
+    "description": "",
+    "points": 6,
+    "skills": [
+      { "level": 3, "name": "survival" },
+      { "level": 3, "name": "gun" },
+      { "level": 2, "name": "rifle" },
+      { "level": 2, "name": "pistol" },
+      { "level": 1, "name": "dodge" },
+      { "level": 1, "name": "melee" },
+      { "level": 1, "name": "pistol" }
```
Comment (eilaattwood) [inline data/json/professions.json]: Duplicate pistol skill definition?

**Comment by ZhilkinSerg** [comment]
Will you resolve conflicts?

**Comment by I-am-Erk** [comment]
Closing until conflicts resolved and/or OP lets us know they're active again. Please comment if you're active again and would like it reopened.

**Pull request closed without merging**